astropy/visualization/wcsaxes/core.py

```diff
@@ -21,6 +23,8 @@
 
 __all__ = ['WCSAxes', 'WCSAxesSubplot']
 
+MATPLOTLIB_LT_30 = LooseVersion(__version__) < LooseVersion('3.0')
```

Comment:
This is not used anywhere in the diff, is this still needed?

Reply:
Oops, a leftover from a previous approach... Fixed now